ansible/eos_install_config

```diff
@@ -42,26 +45,39 @@ requirements:
 options:
     hostname:
         description: IP or FQDN of the device you want to connect to
-        required: true
+        required: True
```

Comment:
Yes, it´s kind of YAML embedded into docstrings for generating plugin documentation for ansible. We should revert to "true|false" to conform to the specs.

Thanks for the input.